dataland-backend/src/main/kotlin/org/dataland/datalandbackend/model/eutaxonomy/nonfinancials/EuTaxonomyActivity.kt

```diff
@@ -29,9 +26,17 @@ data class EuTaxonomyAlignedActivity(
     val activityName: Activity,
     val naceCodes: List<String>?,
     val share: RelativeAndAbsoluteFinancialShare?,
-    @field:Schema(example = JsonExampleFormattingConstants.SUBSTANTIAL_CONTRIBUTION_CRITIREA)
-    val substantialContributionCriteria: Map<EnvironmentalObjective, BigDecimal>?,
-    @field:Schema(example = JsonExampleFormattingConstants.DNSH_CRITIREA)
-    val dnshCriteria: Map<EnvironmentalObjective, YesNo>?,
+    val substantialContributionToClimateChangeMitigation: BigDecimal?,
+    val substantialContributionToClimateChangeAdaption: BigDecimal?,
+    val substantialContributionToSustainableUseAndProtectionOfWaterAndMarineResources: BigDecimal?,
+    val substantialContributionToTransitionToACircularEconomy: BigDecimal?,
```

Comment:
Should those fields have a percentage in the name?

Reply:
I think they do now.

Reply:
ok